app/javascript/components/users/SignInComponent.tsx

```diff
@@ -1,22 +1,47 @@
 // License: LGPL-3.0-or-later
 import React, {useEffect, useState} from "react";
-import {Formik, Form} from 'formik';
+import { createStyles, Theme, makeStyles, useTheme } from '@material-ui/core/styles';
+import {Formik, Form, FormikErrors ,Field, useFormik} from 'formik';
 import Button from '@material-ui/core/Button';
 import noop from "lodash/noop";
 import usePrevious from 'react-use/esm/usePrevious';
+import Typography from '@material-ui/core/Typography';
+import {spacing}  from '@material-ui/system';
+import MuiButton from "@material-ui/core/Button";
+import { styled } from "@material-ui/core/styles";
+import Grid from '@material-ui/core/Grid';
+import { shadows } from '@material-ui/system';
+import logo from './Images/HoudiniLogo.png';
+
+
+import { CardContent, Link } from '@material-ui/core';
+import InputLabel from '@material-ui/core/InputLabel';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import LockOpenIcon from '@material-ui/icons/LockOpen';
+import Input from '@material-ui/core/Input';
+import Card from '@material-ui/core/Card';
+import CardMedia from "@material-ui/core/CardMedia";
+import TextField from '@material-ui/core/TextField';
+
 
 import useCurrentUserAuth from "../../hooks/useCurrentUserAuth";
 import { SignInError } from "../../legacy_react/src/lib/api/errors";
 import { useIntl } from "../../components/intl";
 import * as yup from '../../common/yup';
+import { Email } from '../../legacy_react/src/lib/regex';
+import { autorun } from "mobx";
+import Box from '@material-ui/core/Box';
+import { FormatAlignCenter } from "@material-ui/icons";
+
 
 export interface SignInComponentProps {
 	/**
 	 * An attempt at signing in failed
 	 *
 	 * @memberof SignInComponentProps
 	 */
-	onFailure?:(error:SignInError) =>  void;
+  onFailure?:(error:SignInError) =>  void;
```

Comment:
These two lines are functionally, exactly the same BUT git thinks they've changed because the whitespace around them has changed. `yarn eslint --fix` before adding your files and committing your changes will generally put all the files into a consistent style.